Assets/HoloToolkit-Examples/Input/Scripts/FocusedObjectTest.cs

```diff
@@ -41,6 +41,7 @@ public void OnFocusExit()
             cachedMaterial.SetColor("_Color", originalColor);
         }
 
+        //Destroy object once collision occurs
```

Comment:
We don't normally document known unity messages and methods

Reply:
I agreed that some of the comments are too redundant but I can't really understand the whole part of code, that is why I could only managed to add some of the minor and comments which seems to be meaningless haha.